client/my-sites/plans/trials/trial-acknowledge/trial-plan.tsx

```diff
@@ -8,7 +8,7 @@ import clockIcon from 'calypso/assets/images/jetpack/clock-icon.svg';
 interface TrialPlanProps {
 	subtitle: ReactNode;
 	supportingCopy: ReactNode;
-	trialLimitations: string[];
+	trialLimitations?: string[];
```

Comment:
This is primarily because I do not know what are the trial limitations of Entrepreneur Trial to put underneath for users to be aware of.